src/app/components/action-button/index.html

```diff
@@ -31,4 +31,162 @@
+            </p>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-success">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-on-page.png" style="width: 100%;" />
+			</div>
+        </stache-column>
+    </stache-row>
+    
+    <header><div class="sky-subsection-heading">When not to use action buttons</div></header>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Do not use a single action button by itself. Action buttons should not serve as a primary call-to-action in a page where many other actions are available to the user.
+            </p>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-danger">
+				<img src="~/assets/img/guidelines/actionbutton/single-action-button-dont.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Do not use actions buttons to present different methods for completing a task when the user has not initated the task yet. Action buttons should not compete against other content or actions for the user’s attention.
+            </p>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-danger">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-on-workspace-dont.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Do not use actions buttons when the consequential differences between the choices are a small minority of the scale of the task, or when the choice would not be made at the beginning of the task. Use a different selection input and progressive disclosure to reveal additional content, if needed.
+            </p>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-danger">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-inside-long-dialogue-dont.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <header><div class="sky-section-heading">Workflow placement and behaviors</div></header>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Use a set of action buttons inside a modal when no page already exists, or the remaining bulk of the task after the action button decision takes place inside a modal. This is common when creating a new record that only requires a standard modal during different versions of its 'Add' process.
```

Comment:
This is definitely one of the trickier distinctions to make. I think perhaps an easier angle to take might be questioning if it makes sense for a page/record to exist before the _*end*_ of the larger workflow.

If it makes no sense to create a page until the workflow is finished (most "Add" workflows operate this way) and the interactions are all performed in modal dialogues, than the action buttons should also use a modal container.

If there _*is*_ a page before the workflow finishes, then the action buttons should probably live on that page. The cases where a page already exist for a decision like this should be less common, but we know several examples. New lists in RE NXT are one - tabs are considered a "page-level container" in this case. "Add" interactions complicated enough to require a waterfall page are other good examples, but since the waterfall page is an approved SKY pattern still awaiting implementation/documentation, I tried to avoid using it specifically in my guidelines.

This one is tough. I'm open to suggestions here.

Reply:
That sounds good.